Pop-Grupa-A-Frontend/src/components/clusterNode/ClusterNodeList.jsx

```diff
@@ -38,11 +38,20 @@ class ClusterNodeList extends Component {
             );
     }
 
+    submitCluster() {
+        submitClusterNode(this.state.chosenClusterNodeId)
+            .then(response =>{
+                alert(response.message);
+                this.setState({
+                    listitems : this.getClusters()
+                })
```

Comment:
Można zrobić rozróżnienie w zależności od response.status, jeżeli submit się nie powiedzie to chyba nie trzeba robić setState.